src/IntDict.elm

```diff
@@ -990,16 +1012,21 @@ toList dict =
 -}
 fromList : List ( Int, v ) -> IntDict v
 fromList pairs =
-    List.foldl (uncurry insert) empty pairs
+    List.foldl (\( a, b ) -> insert a b) empty pairs
 
 
 
 -- STRING REPRESENTATION
 
 
 {-| Generates a string representation similar to what `toString`
-generates for `Dict`.
+generates for `Dict`. You must provide a function to convert
+your value type into a string.
 -}
-toString : IntDict v -> String
-toString dict =
-    "IntDict.fromList " ++ Basics.toString (toList dict)
+toString : IntDict v -> (v -> String) -> String
+toString dict valueToStr =
+    let
+        pairToStr (k,v) =
+            "(" ++ String.fromInt k ++ ", \"" ++ valueToStr v ++ "\")"
```

Comment:
Is there really no function on `elm/core` that can convert pairs for us? Why again can't we just use [`Debug.toString`](https://github.com/elm/core/blob/e6b1d6681fb5743b4bc2548173fb829c5311bf8f/src/Debug.elm#L36) any longer?

Reply:
https://alpha.elm-lang.org/packages/elm/core/latest/Debug#toString

Reply:
Thanks for clarifying.

Reply:
no problem. Sorry I was so brief in my reply—I was typing on my phone.